packages/digit_components/lib/widgets/digit_dialog.dart

```diff
@@ -89,9 +94,7 @@ class DigitDialog extends StatelessWidget {
               children: <Widget>[
                 if (options.primaryAction != null)
                   Padding(
-                    padding: options.dialogPadding != null
-                        ? options.dialogPadding!
-                        : const EdgeInsets.all(0),
+                    padding: const EdgeInsets.all(0),
```

Comment:
If we don't have the padding remove the padding instead of adding 0 padding

Reply:
fixed